cli/validate_cmd.go

```diff
@@ -113,8 +113,7 @@ walkLoop:
 
 		pth := walker.Path()
 
-		fi, err := os.Stat(pth)
-		if !fi.IsDir() {
+		if fi, err := os.Stat(pth); err != nil || !fi.IsDir() {
```

Comment:
can we use `os.IsNotExist(err)` and if the error is not nil AND not a file-not-found error, the entire command should fail, because that is unexpected, right?